.eslintrc.json

```diff
@@ -19,27 +19,20 @@
                   "layer:platform",
                   "layer:base",
                   "layer:domain",
-                  "layer:template",
-                  "lib:experience",
-                  "lib:i18n",
-                  "lib:ui",
-                  "lib:form"
+                  "layer:template"
```

Comment:
But it's a violation, `domain` shouldn't depend on `template` layer. So basically everything is allowed for this layer.

Reply:
we use import from core into domains `feature` definitions. So we can't omit platform, and template import we use for picking import. We can create picking exception but we can't omit platform until we move out orchestrator